libs/pong/get.js

```diff
@@ -102,21 +101,36 @@ export function createPongGetHandler(client, coder, env) {
             if (v === null || v?.[0] === null) {
               return [p, null];
             }
-            const [{ contents, clickUrl, impressionUrl }] = v;
-            return [
-              p,
-              {
-                status: "success",
-                copy: he.decode(
-                  contents?.[0]?.data?.title ||
-                    contents?.[0]?.data?.cttitle ||
-                    "This is an ad without copy?!"
-                ),
-                image: coder.encodeAndSign(contents[0]?.data?.imageUrl),
-                click: coder.encodeAndSign(clickUrl),
-                view: coder.encodeAndSign(impressionUrl),
-              },
-            ];
+            if (p === "banner") {
+              const [{ contents, clickUrl, impressionUrl }] = v;
+              return [
+                p,
+                {
+                  status: "success",
+                  copy: he.decode(
+                    contents?.[0]?.data?.title || "This is an ad without copy?!"
+                  ),
+                  image: coder.encodeAndSign(contents[0]?.data?.imageUrl),
+                  click: coder.encodeAndSign(clickUrl),
+                  view: coder.encodeAndSign(impressionUrl),
+                },
+              ];
+            } else if (p === "topBanner") {
+              const [{ contents, clickUrl, impressionUrl }] = v;
+              const { colors, cta } = contents?.[0]?.data?.customData || {};
+              return [
+                p,
+                {
+                  status: "success",
+                  copy: he.decode(contents?.[0]?.data?.title || ""),
+                  cta: he.decode(cta || "No CTA"),
```

Comment:
This means all top placements will have a CTA, or else "No CTA" is shown?

Reply:
yes